nbconvert/exporters/base.py

```diff
@@ -97,20 +100,14 @@ def get_exporter(name, config=get_config()):  # noqa
         name = "notebook"
 
     try:
-        exporter = entrypoints.get_single("nbconvert.exporters", name).load()
+        exporters = importlib_metadata.entry_points()["nbconvert.exporters"]
```

Comment:
this loads _every single entry_point_, and can take _seconds_: indeed, i think it throws warnings on newer pythons. 

Using `entry_points(group="nbconvert.exporters.script")` will 

- https://github.com/jupyterlab/jupyterlab_server/blob/main/jupyterlab_server/translation_utils.py#L94